tutorial.md

```diff
@@ -0,0 +1,262 @@
+# Tutorial: Embedding Taichi in C++ application
+
+As Python becomes the de facto language for a wide variety of areas like scientific computation, image processing and deep learning, deploying alogirthms written in Python in production is now a common challenge for many industries. Taichi's Python frontend provides easy-to-write syntax which makes writing performant parallel code really simple. And Taichi also offers a runtime library (TiRT) and C interface so that kernels written in Python can be embedded into a real application. In this tutorial, we'll walkthrough the steps to deploy a Taichi program in an C++ application.
```

Comment:
```suggestion
Taichi's Python frontend provides easy-to-write syntax which greatly simplifies the process of writing performant parallel computing code, but deploying these alogirthms written in Python into production can be yet another challenge. To bridge the gap between Python and industry, Taichi offers a runtime library (TiRT) with C interface so that kernels written in Python can be embedded into a real application. In this tutorial, we'll walkthrough the steps to deploy a Taichi program in an C++ application.
```